registry/accessing-the-registry.adoc

```diff
@@ -32,13 +32,15 @@ the user must have the `registry-viewer` role. To add this role:
 $ oc policy add-role-to-user registry-viewer <user_name>
 ----
 
-* For writing or pushing images, for example when using the `podman push` command,
-the user must have the `registry-editor` role. To add this role:
+* For writing or pushing images, for example when using the `podman push` command:
```

Comment:
You could avoid nesting these with some restructuring. ISG allows substeps down to two levels, though.